test/e2e/mixed_placement_test.go

```diff
@@ -373,3 +375,164 @@ var _ = Describe("mixed ClusterResourcePlacement and ResourcePlacement negative
 		})
 	})
 })
+
+var _ = Describe("mixed ClusterResourcePlacement and ResourcePlacement positive test cases", Label("resourceplacement"), func() {
+	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+	rpName := fmt.Sprintf(rpNameTemplate, GinkgoParallelProcess())
+	workNamespaceName := fmt.Sprintf(workNamespaceNameTemplate, GinkgoParallelProcess())
+	wantSelectedClusters := []string{memberCluster3WestProdName}
+	wantUnscheduledClusters := []string{memberCluster1EastProdName, memberCluster2EastCanaryName}
+
+	Context("coupling CRP and RP using cluster labeling", Ordered, func() {
+		BeforeAll(func() {
+			By("creating work resources")
+			createWorkResources()
+		})
+
+		AfterAll(func() {
+			ensureRPAndRelatedResourcesDeleted(types.NamespacedName{Name: rpName, Namespace: workNamespaceName}, allMemberClusters)
+			ensureCRPAndRelatedResourcesDeleted(crpName, allMemberClusters)
+
+			By("removing labels from member clusters")
+			Eventually(func() error {
+				for _, clusterName := range wantSelectedClusters {
+					mc := &clusterv1beta1.MemberCluster{}
+					if err := hubClient.Get(ctx, types.NamespacedName{Name: clusterName}, mc); err != nil {
+						return fmt.Errorf("failed to get member cluster %s: %w", clusterName, err)
+					}
+
+					if mc.Labels != nil {
+						delete(mc.Labels, workNamespaceName)
+						if err := hubClient.Update(ctx, mc); err != nil {
+							return fmt.Errorf("failed to update member cluster %s: %w", clusterName, err)
+						}
+					}
+				}
+				return nil
+			}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove labels from member clusters")
+		})
+
+		It("picking N clusters with no affinities/topology spread constraints (pick by cluster names in alphanumeric order)", func() {
+			crp := &placementv1beta1.ClusterResourcePlacement{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: crpName,
+					// Add a custom finalizer; this would allow us to better observe
+					// the behavior of the controllers.
+					Finalizers: []string{customDeletionBlockerFinalizer},
+				},
+				Spec: placementv1beta1.PlacementSpec{
+					ResourceSelectors: namespaceOnlySelector(),
+					Policy: &placementv1beta1.PlacementPolicy{
+						PlacementType:    placementv1beta1.PickNPlacementType,
+						NumberOfClusters: ptr.To(int32(1)),
+					},
+					Strategy: placementv1beta1.RolloutStrategy{
+						Type: placementv1beta1.RollingUpdateRolloutStrategyType,
+						RollingUpdate: &placementv1beta1.RollingUpdateConfig{
+							UnavailablePeriodSeconds: ptr.To(2),
+						},
+					},
+				},
+			}
+			Expect(hubClient.Create(ctx, crp)).To(Succeed(), "Failed to create CRP %s", crpName)
+		})
+
+		It("should update CRP status as expected", func() {
+			crpStatusUpdatedActual := crpStatusUpdatedActual(workNamespaceIdentifiers(), wantSelectedClusters, nil, "0")
```

Comment:
how can we guarantee that we alwasy select this cluster? IIRC, we just did pick 1, it's up to the pick algorithm. Why not just use pick fix to ping it?

Reply:
does our API specify that "with no affinities/topology spread constraints we will pick by cluster names in alphanumeric order"? If not, then this is implementation detail which we can change in the future. As you pointed out, pick the first according to alphabetic is probably not the best.